api/src/routes/programme-degree-route.js

```diff
@@ -0,0 +1,211 @@
+const express = require("express");
+const router = express.Router();
+const LOGGER = require("../common/logger");
+const ProgrammeDegree = require("../models/programme-degree");
+
+// GET All programme degrees
+/**
+ * @swagger
+ * path:
+ *  /programmedegree:
+ *    get:
+ *      tags: [ProgrammeDegree]
+ *      summary: Get all programme degrees
+ *      responses:
+ *        "200":
+ *          description: Array of programme degree objects
+ *          content:
+ *            application/json:
+ *              schema:
+ *                type: array
+ *                items:
+ *                  $ref: '#/components/schemas/ProgrammeDegree'
+ *        "204":
+ *           description: No programme degrees found
+ *        "400":
+ *           description: Error message
+ * 
+ */
+router.get("/", async (req, res) => {
+  ProgrammeDegree.find({}, (err, programmeDegrees) => {
+    if (err) {
+      LOGGER.info("Database error thrown");
```

Comment:
Can you change this to LOGGER.error(err);

Reply:
done